### PR TITLE
user/tofi: new package

### PR DESCRIPTION
**Pull request by jabuxas**

### CODE REVIEW

File: contrib/tofi/template.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
+pkgname = "tofi"
+pkgver = "0.9.1"
+pkgrel = 0
+build_style = "meson"
+hostmakedepends = ["meson", "pkgconf", "scdoc"]
+makedepends = [
+    "wayland-devel",
+    "musl-fts-devel",
+    "freetype-devel",
+    "cairo-devel",
```
Comment (nekopsykose) [inline contrib/tofi/template.py:10]: sort

File: contrib/tofi/template.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    "pango-devel",
+    "wayland-protocols",
+    "libxkbcommon-devel",
+    "linux-headers",
+]
+pkgdesc = "Tiny wayland dynamic menu"
```
Comment (nekopsykose) [inline contrib/tofi/template.py:16]:
```suggestion
pkgdesc = "Wayland dynamic menu"
```

**Comment by jabuxas** [comment]
for some reason post_install fails when installing license..

**Comment by nekopsykose** [comment]
> for some reason post_install fails when installing license..

because the meson.build already installs it to the same place :)

so when you do it it says it already exists

**Comment by jabuxas** [comment]
yea makes sense, thats what i thought as well. though this is the first package i do that already does that automatically.. makes u wonder why other dont

**Comment by nekopsykose** [comment]
some do but not to the exact same folder 😆

you should also be putting these random things in user/ instead

**Comment by jabuxas** [comment]
on the premise that im a new contributor or is there a deeper reasoning? im not against it at all, in fact now that i see, #2874  should probably go to user

**Comment by nekopsykose** [comment]
on the divine premise of 'seems correct'

**Comment by jabuxas** [comment]
lol haha

**Comment by jabuxas** [comment]
id imagine this would be a contrib since fuzzel is contrib as well but doesnt really change much

**Comment by nekopsykose** [comment]
fuzzel is fluffy and very pettable though

**Comment by jabuxas** [comment]
lol

**Pull request merged**